Source/ThirdParty/ANGLE/src/libANGLE/Framebuffer.h

```diff
@@ -162,6 +169,13 @@ class FramebufferState final : angle::NonCopyable
     FramebufferAttachment mDepthAttachment;
     FramebufferAttachment mStencilAttachment;
 
+#if ANGLE_WEBKIT_EXPLICIT_RESOLVE_TARGET_ENABLED
+    // GL_WEBKIT_explicit_resolve_target;
+    DrawBuffersVector<FramebufferAttachment> mColorResolveAttachments;
+    FramebufferAttachment mDepthResolveAttachment;
+    FramebufferAttachment mStencilResolveAttachment;
+#endif
```

Comment:
To address Kimmo's comment, instead of adding an extra slot to `FramebufferAttachment`, add separate storage for the resolve target attachment